js/src/personal/amanda/Main.page.tsx

```diff
@@ -1,55 +1,133 @@
-import { Container, Divider, Pill, Title, Text } from '@mantine/core'
+import {
+  Anchor,
+  Container,
+  Image,
+  Pill,
+  Space,
+  Title,
+  Text,
+  ScrollArea,
+} from '@mantine/core'
 import styles from './Amanda.module.css'
 import { Contact } from './Contact.tsx'
+import { LightDarkButton } from './LightDarkButton.tsx'
 import { Navbar } from './Navbar.tsx'
-import { Section } from './Section.tsx'
+import { Section } from './Types.tsx'
 import data from './Data.json'
 
 export function MainPage() {
   const info = data.info as Section[]
   return (
-    <Container
-      size="responsive"
-      className={styles.mainPage}
-    >
-      <Container className={styles.leftSide}>
-        <Container className={styles.intro}>
-          <Title>{'Amanda Ruan'}</Title>
-          <Text>{'Software Engineer'}</Text>
-          <Contact />
-        </Container>
-        <Navbar />
+    <>
+      <Container
+        className={styles.lightDark}
+        mt="1rem"
+      >
+        <LightDarkButton />
       </Container>
-      <Container className={styles.content}>
-        {info.map((section) => (
+      <Container
+        size="responsive"
+        p="5rem"
+        pb="0"
+        className={styles.mainPage}
+      >
+        <Container className={styles.leftSide}>
           <Container
-            key={section.id}
-            id={section.title}
+            mb="2rem"
+            p="0"
           >
-            <Title>{section.title}</Title>
-            <Text>{section.description}</Text>
-            {section.subsections.map((proj) => (
-              <Container
-                key={proj.id}
-                className={styles.project}
+            <Title size="5rem">{'Amanda Ruan'}</Title>
+            <Text size="1.5rem">{'Software Engineer'}</Text>
+            <Text
+              size="1.5rem"
+              mt="1rem"
+            >
+              {'CS Student @ RPI'}
+            </Text>
+          </Container>
+          <Contact />
+          <Space h="xl" />
+          <Navbar />
+        </Container>
+        <ScrollArea
+          className={styles.content}
+          type="scroll"
+          offsetScrollbars
+        >
+          {info.map((section) => (
```

Comment:
nit: if we are using .map with .map with .map, it seems like a good candidate to refactor into smaller components